isminet/clients/base.py

```
@@ -1,168 +1,182 @@
+        """Handle error response."""
+        error_msg = response.text
+        try:
+            error_data = response.json()
+            if "meta" in error_data and "msg" in error_data["meta"]:
+                error_msg = error_data["meta"]["msg"]
+        except Exception:
+            pass
+
+        if response.status_code == 401:
+            raise AuthenticationError(error_msg)
+        elif response.status_code == 403:
```

Ruff (0.8.2)

Unnecessary `elif` after `raise` statement

Remove unnecessary `elif`

(RET506)